### PR TITLE
Add VarBranchingPriority

### PR DESCRIPTION
**Pull request by laradicp**
fix https://github.com/atoptima/Coluna.jl/issues/521#issuecomment-836709967

### CODE REVIEW

File: test/MathOptInterface/MOI_wrapper.jl [hunk 1]
```diff
@@ -39,6 +39,23 @@ end
...
+        Coluna.Optimizer,
+        "params" => CL.Params(solver=ClA.SolveIpForm()),
+        "default_optimizer" => GLPK.Optimizer
+    )
+    model = BlockModel(coluna, direct_model=true)
+    @variable(model, x)
+    @constraint(model, x <= 1)
+    @objective(model, Max, x)
+    optimize!(model)
+
```
Comment (guimarqu) [inline test/MathOptInterface/MOI_wrapper.jl:53]: Can you try something like `MOI.set(model, Coluna.VarBranchingPrority(), x, 2)` ? The `set method` is not covered.

File: test/MathOptInterface/MOI_wrapper.jl [hunk 2]
```diff
@@ -39,6 +39,23 @@ end
...
+    )
+    model = BlockModel(coluna, direct_model=true)
+    @variable(model, x)
+    @constraint(model, x <= 1)
+    @objective(model, Max, x)
+    optimize!(model)
+    
+    for (moi_index, varid) in backend(model).varids
+        @test MOI.get(backend(model), Coluna.VarBranchingPriority(), moi_index) == 1
+    end
```
Comment (guimarqu) [inline test/MathOptInterface/MOI_wrapper.jl:56]: I think you can just do :

```suggestion
    @test MOI.get(model, Coluna.VarBranchingPriority(), x) == 1
```

File: test/MathOptInterface/MOI_wrapper.jl [hunk 3]
```diff
@@ -39,6 +39,23 @@ end
     @test MOIU.supports_default_copy_to(OPTIMIZER, true)
 end
 
+@testset "branching_priority" begin
+    coluna = JuMP.optimizer_with_attributes(
+        Coluna.Optimizer,
+        "params" => CL.Params(solver=ClA.SolveIpForm()),
+        "default_optimizer" => GLPK.Optimizer
+    )
+    model = BlockModel(coluna, direct_model=true)
```
Comment (guimarqu) [inline test/MathOptInterface/MOI_wrapper.jl:48]: Can you also try with `direct_model = false`?

File: test/MathOptInterface/MOI_wrapper.jl (hunk 2 above)
Comment (laradicp) [reply test/MathOptInterface/MOI_wrapper.jl:56]: No, it doesn't. Should `x` be a `MOI.VariableIndex`? It appears to be a `JuMP.VariableRef`.

**Comment by guimarqu** [comment]
Great, I'll merge it after the release of BlockDecomposition.

**Comment by laradicp** [comment]
This last commit is due to the change in `function MOI.get(dest::MOIU.UniversalFallback, attribute::VarBranchingPriority, vi::MOI.VariableIndex)`. Now `direct_model=false` also returns the default value when `BlockDecomposition.branchingpriority` is called.

**Review by guimarqu — APPROVED** [review]

**Pull request merged**